### ISSUE TITLE
more robust handling of CTF's

### ISSUE THREAD

**Issue by mushinlogit** [unfetter-discover/unfetter#855]
may need to handle different cyber threat frameworks (CTF) during ingest and in the UI.  Generate and save a CTF in mongo?

**Comment by mushinlogit** [comment]
waiting on updated spreadsheet, unassigning for now

**Comment by mushinlogit** [comment]
made several improvements, will probably need to create a new issue to track work in a future sprint.  Closing this for now
